webapp/model/commentModel.js

```diff
@@ -0,0 +1,48 @@
+sap.ui.define(
+  [
+    "sap/ui/model/json/JSONModel",
+    "nadzeya/kireyenka/app/model/baseModel",
+    "sap/base/util/deepClone",
+    "nadzeya/kireyenka/app/helper/helper",
+    "nadzeya/kireyenka/app/model/userModel",
+  ],
+  function (JSONModel, baseModel, deepClone, helper, userModel) {
+    "use strict";
+
+    const oBaseModel = deepClone(baseModel);
+
+    return Object.assign(oBaseModel, {
+      _model: new JSONModel("/data/comments.json"),
+
+      /**
+       * Create comment(s).
+       *
+       * @param {(Object | Object[])} uPayload - New comment(s) payload.
+       *
+       * @returns {(Object | Object[])} - New comment(s) full data.
+       */
+      create: function (uPayload) {
+        let bIsPayloadArray = true;
+
+        if (!Array.isArray(uPayload)) {
+          bIsPayloadArray = false;
+          uPayload = [uPayload];
+        }
+
+        const aComments = this.getData();
+
+        uPayload.forEach((oNewCommentData) => {
+          oNewCommentData.Id = helper.getTimestampString();
+          oNewCommentData.Author = userModel.getData().Name;
+          oNewCommentData.Date = helper.getJsonDateStringFromDate(new Date());
+
+          aComments.push(oNewCommentData);
+        });
+
+        this.setData(aComments);
+
+        return bIsPayloadArray ? uPayload : uPayload[0];
```

Comment:
Maybe we can remove this variable and just check length property (return uPayload.length > 1 ?  uPayload : uPayload[0])